projects/sandbox/src/app/problem-viewer/problem-viewer.component.html

```diff
@@ -296,7 +296,7 @@ <h2 class="title record-title">
       ></ng-content>
     </ng-container>
 
-    <ul class="form-group header-info">
+    <ul class="form-group header-info" *ngIf="!enableDynamicInfo">
```

Comment:
The temporary variable `enableDynamicInfo` appears in 4 places, thrice with the `@Input` decorator.

It should be changed in one place only (in `wizard.component`) - then it will cascade down to the other places, keeping them all in sync!  It looks like they're out of sync in the screenshot above - as if the variable was changed somewhere where it's an `@Input`.

Reply:
After our meeting and your explanation I am able to see the update.
![image](https://user-images.githubusercontent.com/19560266/194841305-c6d74c57-4f31-4763-9a08-5a5e54cfb20d.png)